test/blackbox-tests/test-cases/rule/synopses/deps.t

```diff
@@ -0,0 +1,38 @@
+Dune should attach synopsis to alias, but should not attach deps synopsis.
```

Comment:
This sentence is hard to understand.

Reply:
I failed to understand motivation for this test, so removed it and replaced in 43059fbd6